python/torch_mlir/dialects/torch/importer/jit_ir/build_tools/abstract_interp_lib_gen.py

```diff
@@ -970,6 +970,20 @@ def aten〇index〇Tensor_hacked_twin〡shape(self: List[int], indices: List[Lis
 def aten〇cat〡shape(tensors: List[List[int]], dim: int = 0) -> List[int]:
     return upstream_shape_functions.cat(tensors, dim)
 
+def aten〇fft_fft〡shape(self: List[int], n: Optional[int] = None, dim: int = -1, norm: Optional[str] = None) -> List[int]:
+    return self
+
+def aten〇fft_fft〡dtype(self_rank: int, self_dtype: int, n: Optional[int] = None, dim: int = -1, norm: Optional[str] = None) -> int:
```

Comment:
Can you add checks thoroughly testing the logic of this implementation?

For reference, see:

https://github.com/llvm/torch-mlir/blob/64f9a0e978d530a2c00d2134e6a01c41cb034b9a/python/torch_mlir/dialects/torch/importer/jit_ir/build_tools/abstract_interp_lib_gen.py#L51-L65

https://github.com/llvm/torch-mlir/blob/64f9a0e978d530a2c00d2134e6a01c41cb034b9a/python/torch_mlir/dialects/torch/importer/jit_ir/build_tools/testing_framework.py#L15

Reply:
On Github I still don't see checks added here

Reply:
Oh my bad I totally missed your comment. Will add.